homeassistant/components/nordpool/quality_scale.yaml

```
@@ -0,0 +1,101 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry:
+    status: exempt
+    comment: |
+      This integration only allows single config entry through manifest
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities doesn't subscribe to events.
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: todo
+  docs-removal-instructions: todo
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  brands: done
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: exempt
+    comment: |
+      No actions.
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      This integration does not require authentication.
+  parallel-updates: todo
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: todo
+  docs-configuration-parameters:
+    status: exempt
+    comment: |
+      No options to configure
+
+  # Gold
+  entity-translations: done
+  entity-device-class: done
+  devices: done
+  entity-category: done
+  entity-disabled-by-default: done
+  discovery:
+    status: exempt
+    comment: |
+      No discovery, cloud service
+  stale-devices:
+    status: exempt
+    comment: |
+      This integration devices (services) will be removed with config entry if needed.
+  diagnostics: done
+  exception-translations:
+    status: exempt
+    comment: |
+      No exceptions to translate
+  icon-translations: done
+  reconfiguration-flow: done
+  dynamic-devices:
+    status: exempt
+    comment: |
+      This integration has fixed devices.
+  discovery-update-info:
+    status: exempt
+    comment: |
+      No discovery
+  repair-issues:
+    status: exempt
+    comment: |
+      This integration doesn't have any cases where raising an issue is needed.
+  docs-use-cases: done
```

```suggestion
  docs-use-cases: todo
```
for me at least.  I couldn't get from the documentation what you can use that data for

---

I discussed this a bit and I do think I am missing some use cases in the documentation. Like it mentions implicitly its for energy prices, but what can one do with energy prices? For whom is this the price to pay? When does this apply to me?

---

I made the change but I don't agree. No one would even look at Nord Pool if you don't know what it is and why you want to use it.
It's not exactly the same as when you're scouting for a device to buy or something as such...

---

Oh I agree that nordpool isn't a device you casually search via the docs, but it is nice that if someone mentions nord pool in a conversation you are able to understand what it does and what that means, because I have heard the name a lot so far, and I still have no clue to whom it applies

---

The only thing I noticed is that the week after we merged this PR the same blue greenish logo is used for internal systems in my local government which made me even more confused 😂

---

Once you guys can approve https://github.com/home-assistant/core/pull/130185 then it's actually going to be useful for a lot of people 🚀